### ISSUE TITLE
include test files in source tarball (aka sdist)

### ISSUE THREAD

**Issue by eighthave** [mwclient/mwclient#182]
I'm working on packaging _mwclient_ for [Debian](https://bugs.debian.org/886826).  Debian packages generally work from the official source tarball. The packaging process should run the test suite, but the _tests/_ subdir is not included in the source tarball.

Please add the test files to _MANIFEST.in_, e.g.:
```
include tests/test_client.py
include tests/test_listing.py
include tests/test_page.py
include tests/test_sleep.py
include tests/test_util.py
```

**Comment by danmichaelo** [comment]
Found that `test/test*.py` is added by default, so I just renamed the `tests` folder to `test`. Will publish a new patch release soon.

**Comment by danmichaelo** [comment]
Published 0.8.7
